### DISCUSSION TITLE
Starrate should change if a touch device is detected

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#19405] [Q&A] [answered]
I tried to find a discussion of this earlier but couldn't find it. As far as I know, the osu!stable for touch devices has its own starrate (?), which is much lower than the usual starrate. I think that when I run osu!lazer on my phone, I should see the current starrate for touch devices.

Is there a plan to implement this system?

**Comment by frenzibyte** [comment]
Related: https://github.com/ppy/osu/issues/5378

**Reply by Moltanicaa** [reply]
It talks about the mod icon, but I write about changing the starrate, because the gaming experience is very different on the touch screen.

**Reply by frenzibyte** [reply]
It's related in that we will add logic to handle touchscreen devices and differentiate them from normal scores, and I'm only mentioning it to be related, not a duplicate of that issue.

**Comment by smoogipoo** [comment, marked as answer]
There's no star rating for TD currently, only a hidden multiplier for PP. There's an open PR for moving it to SR: https://github.com/ppy/osu/pull/16524
